Exiled.Events/Patches/Events/Server/RoundEnd.cs

```diff
@@ -185,6 +191,7 @@ private static IEnumerator<float> Process(RoundSummary roundSummary)
                                 RoundEndedEventArgs roundEndedEventArgs = new((API.Enums.LeadingTeam)leadingTeam, newList, num5);
 
                                 Handlers.Server.OnRoundEnded(roundEndedEventArgs);
+                                Round.IgnoredPlayers.Clear();
```

Comment:
should be place with Map.ClearCache() 
Not here

Reply:
> should be place with Map.ClearCache() Not here

Map doesn't sound like the ideal place to do it though, but ok